nats.go

```diff
@@ -2507,8 +2507,10 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
-// Servers returns the list of known servers, including additional
-// servers discovered after a connection has been established.
+// Servers returns the list of known server urls, including additional
+// servers discovered after a connection has been established.  URLs passed
+// into NATS through the API be preserved in their original form, but any
+// credentials in a URL will never be visible outside of the application.
```

Comment:
I would leave that comment out. Did we decide if we wanted to mark which ones were explicit vs implicit via wrapping in a struct? @kozlovic what do you think?